tester/src/test/scala/spinal/tester/scalatest/ChecksTester.scala

```diff
@@ -528,7 +528,7 @@ class RepeatabilityTester extends AnyFunSuite{
 
   test("Apb3I2cCtrlGraph"){
     val dut = SpinalConfig(defaultClockDomainFrequency = FixedFrequency(50 MHz)).generateVerilog(new Apb3I2cCtrl(configI2C)).toplevel
-    assert(GraphUtils.countNames(dut) == 217)
+    assert(GraphUtils.countNames(dut) == 219)
```

Comment:
Why?

Reply:
Repeatability tester. As far as I can tell this is checking that the same number of signals are generated each time. The error flags added extra signals. This'll likely change again when proper error logic is implemented.

Reply:
Okay, thanks for the explanation :smile: